user_guide/data_management_grids.rst

```diff
@@ -0,0 +1,306 @@
+.. _user-guide-ui-components-grids:
+    
+Grids
+=====
+
+Initially, a grid is an aggregated view of all the records of the same entity. Each row of a grid is one records, each 
+column is one of the grid properties, and thus each cell displays a value of a property for a record. .
+
+For example, you can open a grid of all the Contacts. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_01.png
+
+|
+
+In the example, you can immediately see some major details of our contacts, such as their first and last names, email, 
+phone number, etc.
+
+You can adjust and save grid views, updated and delete grid records, get to the Edit and View pages of any grid record, 
+and even export all the data in the grid as a .csv file.
+
+Get to a Grid
+-------------
+
+There are several ways to get to a grid.
+
+- Select the relevant point in the :ref:`menu bar <user-guide-navigation-menu>`: to get to the *"All Contacts"* grid, 
+  select the *"Contacts"* menu item, to get to the *"All Reports"* grid, click the *"Manage Custom Reports"* menu item, 
+  etc. 
+  
+- Find it in the :ref:`Shortcuts <user-guide-getting-started-shortcuts>`.
+
+- Click the grid link on the :ref:`View page <user-guide-ui-components-view-pages>` 
+  or :ref:`Create/Edit form <user-guide-ui-components-create-pages>` of a record. 
+
+|
+
+.. image:: ./img/data_management/grid/grid_from_view.png
+
+|
+
+.. hint::
+
+    If you have reached a View page or Create/Edit form from a grid, and now click  link to get back to the grid, this 
+    grid will be look in the same as it did when you left it for the View page (the same filters and order applied).    
+
+    
+
+Adjust a Grid
+-------------
+
+.. _user-guide-ui-components-grids-change-table:
+
+Change the Table
+^^^^^^^^^^^^^^^^
+
+Actually, you can change the set of details available in the grid (if permitted), and the order of columns.
+To do so, go to the right corner of the grid, and click the settings icon |IcSettings|.
```

Comment:
Edit: You can change the set of details available in the grid (if permitted) and its order of columns by going to the right corner of the grid and clicking the settings icon |IcSettings|.